src/Layout/ComputationMenu/ComputationMenu.tsx

```diff
@@ -320,6 +320,34 @@ export class ComputationMenu extends React.Component<ComputationMenuProps, {}> {
         );
     };
 
+    /**
+    * Helper function to check if settings have changed.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [asterisks in jsdoc must be aligned](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5596048)